ast/term.go

```diff
@@ -1085,6 +1085,18 @@ func (ref Ref) OutputVars() VarSet {
 	return vis.Vars()
 }
 
+// DropHead drops the first component of this ref.
+func (ref Ref) DropHead() Ref {
```

Comment:
Hmm I'm hesitant to expand the `ast` API for this... I might add a small follow-up to inline it instead.